Content.Shared/_NF/Bank/SharedBankSystem.cs

```diff
@@ -9,7 +9,8 @@ namespace Content.Shared.Bank;
 public enum BankATMMenuUiKey : byte
 {
     ATM,
-    StationATM
+    StationATM,
+    WithdrawATM
```

Comment:
this is unused. since you've created your own BUI and arent integrating this, it appears you are still just using the same key byte for a check. In this instance, I'm using similar code from the ATM as the station administration console, so I simply use a different Key on the entity prototype and the code acts differently. Here it is ok that we re-use ATM, since you have your own BUI you are calling and it makes sense for now.